macros/src/main/scala/gestalt/macros/Optional.scala

```diff
@@ -28,11 +28,14 @@ final class Optional[+A >: Null](val value: A) extends AnyVal {
         Select(tempIdent, "value")
     }
 
-    q"""
+    {
+      import scala.gestalt.options.unsafe
+      q"""
        $tempValDef
        if ($tempIdent.isEmpty) new Optional(null)
-       else new Optional($newBody)
+       else new Optional(${newBody.wrap})
```

Comment:
Usually the conversion is automatic, it's strange we need to add the conversion explicitly.